.ci/wercker.sh

```diff
@@ -82,12 +82,12 @@ no-error-orekit)
   git checkout $SHA_HIPPARCHUS
   mvn -e install -DskipTests
   cd -
-  checkout_from https://github.com/CS-SI/Orekit.git
+  checkout_from https://github.com/Abhishek-kumar09/Orekit.git
```

Comment:
please send PR https://github.com/CS-SI/Orekit/pulls/Abhishek-kumar09

Reply:
Done fix is merged.
https://github.com/CS-SI/Orekit/pull/28

https://github.com/checkstyle/checkstyle/pull/8083/files#diff-8b07316a80f4b78a810d359a8be767cb13f5831e827aaa3b64cc36b6d133c2a0

Reply:
Ok, please their repo and commit where it was merged ( if we used strict commit)

Reply:
Already done.